user-docs/modules/ROOT/pages/fdo-terminology.adoc

```diff
@@ -0,0 +1,43 @@
+= FIDO device onboarding terminology
```

Comment:
Is glossary a better word than terminology?

Reply:
Changed